config/core/configmaps/deployment.yaml

```diff
@@ -44,6 +44,10 @@ data:
     # List of repositories for which tag to digest resolving should be skipped
     registriesSkippingTagResolving: "kind.local,ko.local,dev.local"
 
+    # digestResolutionTimeout is the maximum time allowed for an image's
+    # digests to be resolved.
+    digestResolutionTimeout: "2s"
```

Comment:
Do we have any data backing this up? I.e. do we have historical data or something showing how long a digest usually takes?

Reply:
no, that's why I didn't want to lower it too much without making it configurable (hard to know with total certainty what's a safe value in all environments), but I think 2 second is still _fairly_ generous -- it's only a metadata request. Very happy to bump it a bit higher though if we're concerned.

Reply:
the critical thing here is if the default is bad people can trivially bump it back up (but 60 seconds seems way way too high to me as a default, even with plenty of resolver threads)

Reply:
slack negotiation landed on 10s :). updated